extension-definition-specifications/observed-string-8b1/extension-definition--8b1aa84c-5532-4c69-a8e7-b6170facfd3d.json

```diff
@@ -1,12 +1,12 @@
 {
     "$id": "https://raw.githubusercontent.com/oasis-open/cti-stix-common-objects/main/extension-definition-specifications/observed-string-8b1/extension-definition--8b1aa84c-5532-4c69-a8e7-b6170facfd3d.json",
-    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
     "title": "observed-string-extension-schema",
     "description": "This is the wrapper for the observed-strings extension",
     "type": "object",
     "allOf": [
       {
-        "$ref": "http://raw.githubusercontent.com/oasis-open/cti-stix2-json-schemas/stix2.1/schemas/common/cyber-observerable-core.json"
+        "$ref": "https://raw.githubusercontent.com/oasis-open/cti-stix2-json-schemas/master/schemas/common/cyber-observable-core.json"
```

Comment:
Is there a reason to use the `master` branch instead of `stix2.1`?

Reply:
No - stix2.1 is correct